change/@ni-nimble-components-78757500-8f52-4913-b98c-dfc88b407979.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
>it was causing some/all of the action button/link and close control to be out-of-view when both the title and the button/link text were long

In [this chromatic diff](https://www.chromatic.com/pullrequest?appId=60e89457a987cf003efc0a5b&number=2155&activeElementId=comparison-test-key-63e2b3e0eb6fb7cdb2d44ef5-1200px&view=changes) it looks to me like the baseline is correct but the new build is not (buttons are clipping on their right edge). Am I looking at the wrong diff or misreading Chromatic or something?

Reply:
Looks like the difference between `break-word` and `anywhere` is relevant here (i.e. for `anywhere`, "Soft wrap opportunities introduced by the word break are considered when calculating min-content intrinsic sizes"). Switched (and updated the PR description).